packages/ocaml/ocaml.4.14.1/opam

```diff
@@ -17,7 +17,11 @@ setenv: [
   [OCAML_TOPLEVEL_PATH = "%{toplevel}%"]
 ]
 build: ["ocaml" "%{ocaml-config:share}%/gen_ocaml_config.ml" _:version _:name]
-build-env: CAML_LD_LIBRARY_PATH = ""
+build-env: [
+  [CAML_LD_LIBRARY_PATH = ""]
+  [LSAN_OPTIONS = "detect_leaks=0,exitcode=0"]
+  [ASAN_OPTIONS = "detect_leaks=0,exitcode=0"]
+]
```

Comment:
this will trigger a rebuild for all the compiler and the addition seems rather weird. Could you avoid changing the `ocaml` packages?

Reply:
I don't think so. This is following @dra27's advice here: https://github.com/ocaml/opam-repository/pull/21667#issuecomment-1205234468

If that's too much trouble, we can restrict support to the latest ocaml 5 beta release and move on from there with future versions.

Reply:
> If that's too much trouble, we can restrict support to the latest ocaml 5 beta release and move on from there with future versions.

in this case I'd rather we do that